bugbug/labels.py

```diff
@@ -42,4 +42,6 @@ def get_all_bug_ids():
 
             bug_ids.update([int(row["bug_id"]) for row in reader])
 
+    bug_ids.update(commit["bug_id"] for commit in repository.get_commits())
+
     return list(bug_ids)
```

Comment:
What is the length of this list before/after your changes? I want to make sure we're not adding too many bugs here

Reply:
you're right, previously 12967 bugs and after changes 242435 bugs, so what should we do?
PS. The downloading is very slow and takes like 3 hours approx. :(

Reply:
A possible option is to restrict commits to the past 2 years (or 2 years and a half).
Something like:
```Python
start_date = datetime.now() - relativedelta(years=2, months=6)
bug_ids.update(commit["bug_id"] for commit in repository.get_commits() if commit["bug_id"] and dateutil.parser.parse(commit['pushdate']) <= start_date)
```

What is the number of bugs then?

Reply:
179964

Reply:
Can you check how many of these bugs are not already there in the bugs DB?

Reply:
If the two sets are disjoint, we can't really add 180000 bugs, but if the two sets share a lot of bugs maybe the 180000 will become way less.

Reply:
166000 are not there in bugs db.(only 13000 are)

Reply:
Oh wow, I was not expecting such a big number!

Reply:
Did you use my snippet as-is? There was an error (`<=` instead of `>=`). If you fix that, the number is 23k.

Reply:
I had a feeling that it should be >= but i used the snippet as is :(